CHANGELOG.md

```diff
@@ -24,6 +24,9 @@ Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
 - Updated grunt-eslint to `^13.0.0`.
 - Moved eslint config file to home directory.
 - Moved jQuery download to package.json.
+- Updated grunt-banner to `^0.4.0` and updates banner ascii art and copy.
+- Changed bower.json authors array field to single field author,
```

Comment:
Also fixed in https://github.com/cfpb/cfgov-refresh/commit/8dbd55c246b84f0cc5e76a416224a19dba55e815